src/scss/components/_mainImage.scss

```diff
@@ -12,7 +12,7 @@
     box-shadow: 0 0 15px 10px #505050;
```

Comment:
Wszystko spoko tylko mógłbyś zmienić kolejność styli żeby były bardziej uporządkowane mniej więcej w ten sposób.
`   width: 250px;
    height: 350px;
    position: relative;
    top: 50%;
    left: 25%;
    background-color: silver;
    border: 2px #404040;
    transform: translate(-50%, -50%);
    -moz-box-shadow: 0 0 10px 10px #505050;
    -webkit-box-shadow: 0 0 10px 10px #505050;
    box-shadow: 0 0 15px 10px #505050;`

Reply:
Style będą do poprawki, na razie to taka wersja na bardzo szybko. Piotrek wspominał, że task na to będzie i że w ogóle niepotrzebnie się tym zajmowałem, ale już nie chciałem kasować, jak coś miałem.
Skasowałem już brancha, także myślę, że w tym tasku ze stylami się to poprawi :)